http-client/src/main/java/io/airlift/http/client/jetty/DefaulHttpClientLogger.java

```diff
@@ -88,6 +89,12 @@ public void close()
         asyncAppender.stop();
     }
 
+    @Managed
+    public int getNumberOfElementsInQueue()
```

Comment:
This method name seems overly verbose. How about `getQueueSize()`, since that will match the config name.